client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -481,13 +484,13 @@ class EmailProvidersComparison extends Component {
 				? formatCurrency( gSuiteProduct?.cost ?? null, currencyCode )
 				: getAnnualPrice( gSuiteProduct?.cost ?? null, currencyCode );
 
-		const discount = this.getAvailableDiscountForGoogle(
+		const discount = this.getAvailableDiscountForGoogle( {
 			currencyCode,
 			isEligibleForFreeTrial,
 			gSuiteProduct,
 			productIsDiscounted,
-			standardPrice
-		);
+			standardPrice,
+		} );
```

Comment:
Not a change request, but curiosity! What's the benefit of encapsulated parameters?

Reply:
Same question 😀.

Reply:
My (admittedly strong) personal preference is to use encapsulated parameters in any situation where I expect the parameter list to grow beyond 1 parameter. My thinking is that this makes it easier to add parameters over time in a way that is seamless, and generally requires fewer changes to callers (especially when you get to adding a second and later parameters). I also like it because it forces me to treat the function more like a component that receives props - it helps to make more dependencies injected/supplied and makes it easy to refactor the logic into a React component if needed.